onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -75,9 +75,10 @@ def _get_qrange_for_qType(qType, reduce_range=False):
 
 
 class ONNXQuantizer:
-    def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType, input_qType, quantization_params,
+    def __init__(self, model_path, per_channel, reduce_range, mode, static, weight_qType, input_qType, quantization_params,
```

Comment:
do you need to change code here? how about just call save_model_to_file for each quantization method?

Reply:
> do you need to change code here? how about just call save_model_to_file for each quantization method?

Here changing model-> model_path is used for shape inference. Since onnx.shape_inference.infer_shapes() takes in a model path and it won't return a model as before. So if we want to get the inferred_model, we need to load from the original path, so I thought we need to take in a model_path for ONNXQuantizer anyways. I can think about if there's any better solutions though.